mito-ai/src/Extensions/AiChat/ChatMessage/ChatMessage.tsx

```diff
@@ -205,13 +208,22 @@ const ChatMessage: React.FC<IChatMessageProps> = ({
                 }
             })}
             {editable && 
-                <button
-                    className="message-start-editing-button"
-                    onClick={handleEditClick}
-                    title="Edit message"
-                >
-                    <PencilIcon />
-                </button>
+                <div className="message-action-buttons">
+                    <button
+                        className="message-start-editing-button"
+                        onClick={handleEditClick}
+                        title="Edit message"
+                    >
+                        <PencilIcon />
+                    </button>
+                    <button
+                        className="message-delete-button"
+                        onClick={() => onDeleteMessage?.(messageIndex)}
```

Comment:
In practice we're always setting an `onDeleteMessage`. So let's just make this non-optional in the component declaration and then rely on the logic we already have here to only display these buttons if it is `editable`

Reply:
The only thing is that the `delete` is a bit weird on a regular user input message because it just removes the user input, but not the associated response. I don't think it's a huge deal for now, but maybe we can just open an issue to remind ourselves to fix this at some point.